tequilapi/endpoints/sessions.go

```diff
@@ -111,7 +111,7 @@ func sessionToDto(se session.Session) SessionDTO {
 		ProviderID:      string(se.ProviderID.Address),
 		ServiceType:     se.ServiceType,
 		ProviderCountry: se.ProviderCountry,
-		DateStarted:     se.Started.Format("2018-10-29 16:22:05"),
+		DateStarted:     se.Started.Format("2006-01-02 15:04:05"),
```

Comment:
It looked okey from the first sight. And then I started thinking (maybe this is the real problem 😄 )
What timezone this format uses? What timezone we use when we set actual session start time?
Maybe its better to use one of predefined ISO formats with UTC time zone? In that case session times will be comparable in whole world.
These are not change requests but more like answers we need to answer and to do that maybe we have to look deeper. This looks like good opportunity to do that

Reply:
I've updated the format to RFC3339 which is basicly a stricter ISO 8601.